ci/get_labels/main_test.go

```diff
@@ -0,0 +1,89 @@
+package main_test
+
+import (
+	"encoding/json"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/GoogleCloudPlatform/ops-agent/ci/get_labels"
+)
+
+type Test struct {
+	Name   string
+	Labels main.RespLabelCollection
+}
+
+func TestLabels(t *testing.T) {
+	testTable := []Test{
+		{
+			Name: "empty",
+		},
+		{
+			Name: "release & foo",
+			Labels: main.RespLabelCollection{
+				{
+					Name: "release",
+				},
+				{
+					Name: "foo",
+				},
+			},
+		},
+	}
+
+	for _, test := range testTable {
+		test := test
+		t.Run(test.Name, func(t *testing.T) {
+			testGetLabels(t, test.Labels)
+		})
+	}
+}
+
+func testGetLabels(t *testing.T, expected main.RespLabelCollection) {
+	t.Parallel()
+	server := getHttpClient(t, expected)
+	defer server.Close()
+	labelCollector := main.LabelCollector{
+		Client: server.Client(),
+	}
+
+	actual, err := labelCollector.GetLabels(server.URL + "/GoogleCloudPlatform/ops-agent/pulls/818")
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	assertRespLabelCollection(t, expected, actual)
+}
+
+func assertRespLabelCollection(t *testing.T, expected main.RespLabelCollection, actual main.RespLabelCollection) {
```

Comment:
Maybe `assertRespLabelCollectionEqual` just to convey that you're asserting equality